src/sst.rs

```diff
@@ -248,10 +355,10 @@ impl<'a> EncodedSsTableBuilder<'a> {
         );
 
         self.block_meta.push(block_meta);
-        let checksum = crc32fast::hash(&encoded_block);
-        let total_block_size = encoded_block.len() + std::mem::size_of::<u32>();
+        let checksum = crc32fast::hash(&compressed_block);
+        let total_block_size = compressed_block.len() + std::mem::size_of::<u32>();
```

Comment:
Can a simple test at sst level be added for one of the compression types? I see that the enum and specific compression is under feature flag. Should we start to add more steps in [pr.yaml](https://github.com/slatedb/slatedb/blob/main/.github/workflows/pr.yaml) for different features?

Reply:
Ya, that's a good idea. Should probably add a matrix to pr.yaml to read both uncompressed and at least one compressed type... maybe all 4. Seems excessive, but since they're differently libraries and we don't have a lot of tests, it's probably fine for now to do all 4.

Reply:
Actually, easiest thing to do is probably just parameterize one of the tests to take a compression param and run all compressions through it.

Reply:
Opened #130